fdbclient/ServerKnobs.cpp

```diff
@@ -799,6 +799,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( CC_MIN_DEGRADATION_INTERVAL,                         120.0 );
 	init( ENCRYPT_KEY_PROXY_FAILURE_TIME,                        0.1 ); if ( isSimulated ) ENCRYPT_KEY_PROXY_FAILURE_TIME = 1.0 + deterministicRandom()->random01();
 	init( CC_DEGRADED_PEER_DEGREE_TO_EXCLUDE,                      3 );
+	init( CC_DEGRADED_PEER_DEGREE_TO_EXCLUDE_MIN,                  1 );
```

Comment:
The naming is unfortunate here. It'll be hard to change `CC_DEGRADED_PEER_DEGREE_TO_EXCLUDE` to reflect it's an upper bound. So the best I could think of was to have a similar version with MIN suffix. Happy to change if there's a better name.